fedimint-client/src/lib.rs

```diff
@@ -1810,6 +1824,18 @@ impl ClientBuilder {
         )
     }
 
+    pub fn with_meta_source(&mut self, meta_source: impl MetaSource + 'static) {
+        Arc::get_mut(&mut self.meta_manager)
+            // Self::from_existing is private, so should never happen
+            .expect("with_meta_source when meta manager is non unique")
+            .add_source(meta_source);
+    }
+
+    /// NOTE: [`Self::with_meta_source`] may fail if this meta_manager is not unique.
+    pub fn with_meta_manager(&mut self, meta_manager: Arc<MetaManager>) {
```

Comment:
this method allows you to even more default meta sources.

For example you can wrap the default config source to modify some meta values.